src/containers/item-details/item-details.tsx

```diff
@@ -26,7 +26,7 @@ const RenderedItem = () => {
 
   const isAuthor = address === selectedItem?.author?.address;
 
-  const priceLabel = isOwner ? 'You own this NFT' : 'Buy it now for';
+  const priceLabel = isOwner ? 'You own this NFT of' : 'Buy it now for';
```

Comment:
Do we need the 'of' word?

Reply:
Yeah, because when you are the owner, you can't see the item price and the final message is "You own this NFT of 0.001 MNL"